frontend/app/views/spree/checkout/payment/_gateway.html.erb

```diff
@@ -1,27 +1,28 @@
 <%= image_tag 'credit_cards/credit_card.gif', id: 'credit-card-image' %>
 <% param_prefix = "payment_source[#{payment_method.id}]" %>
 
-<div class="field field-required">
+<div class="field field-required card_name">
   <%= label_tag "name_on_card_#{payment_method.id}", t('spree.name_on_card') %>
-  <%= text_field_tag "#{param_prefix}[name]", "#{@order.billing_firstname} #{@order.billing_lastname}", { id: "name_on_card_#{payment_method.id}", autocomplete: "cc-name" } %>
+  <%= text_field_tag "#{param_prefix}[name]", "#{@order.billing_firstname} #{@order.billing_lastname}", { id: "name_on_card_#{payment_method.id}", autocomplete: "cc-name", class: 'cardName' } %>
 </div>
 
-<div class="field field-required" data-hook="card_number">
```

Comment:
I think we should keep both when the data-hook is already there otherwise we'll break stores that rely on those data-hook

Reply:
Fair enough. Do you think we still should add those classes **and** keep the `data-hook`s?